README.md

```diff
@@ -37,26 +37,34 @@ make build
 
 ## Deploy
 
-This is still a WIP as I would like to streamline the certificate creation experience via something like a Helm Chart.
+There are two types of Webhook Admission controllers in Kubernetes 1.9.
+* ValidatingAdmissionWebhook
+* MutatingAdmissionWebhook
 
-## Explanation on the CAs/Certs/Keys
+Enable the relevant Kubeneretes Admission controller by adding to following `--admission-control` and restarting kube-apiserver. See the relevant [docs](https://kubernetes.io/docs/admin/extensible-admission-controllers/#external-admission-webhooks)
```

Comment:
typo: Kubeneretes should be Kubernetes
typo: missing `.` at the end of the sentence?